spec/lib/hamster/nested/construction_spec.rb

```diff
@@ -4,6 +4,9 @@
 require "set"
 
 describe Hamster do
+
+  Struct.new("Customer", :name, :address)
```

Comment:
OK, sorry, I see it's not extraneous now. I moved it to `spec_helper.rb`, though.